consensus/src/pending_order_votes.rs

```diff
@@ -271,4 +268,98 @@ mod tests {
         assert!(!pending_order_votes.has_enough_order_votes(&li1));
         assert!(!pending_order_votes.has_enough_order_votes(&li2));
     }
+
+    #[test]
+    fn order_vote_aggregation_with_unverified_votes() {
+        ::aptos_logger::Logger::init_for_testing();
+
+        let (signers, verifier) = random_validator_verifier(5, Some(3), false);
+        let mut pending_order_votes = PendingOrderVotes::new();
+        let mut partial_signatures = PartialSignatures::empty();
+        let qc = QuorumCert::dummy();
+
+        // create random vote from validator[0]
+        let li = random_ledger_info();
+        let vote_0 = OrderVote::new_with_signature(
+            signers[0].author(),
+            li.clone(),
+            signers[0].sign(&li).expect("Unable to sign ledger info"),
+        );
+        partial_signatures.add_signature(signers[0].author(), vote_0.signature().clone());
+
+        let vote_1 = OrderVote::new_with_signature(
+            signers[1].author(),
+            li.clone(),
+            signers[1].sign(&li).expect("Unable to sign ledger info"),
+        );
+        partial_signatures.add_signature(signers[1].author(), vote_1.signature().clone());
+
+        let vote_2 = OrderVote::new_with_signature(
+            signers[2].author(),
+            li.clone(),
+            bls12381::Signature::dummy_signature(),
+        );
+
+        let vote_3 = OrderVote::new_with_signature(
+            signers[3].author(),
+            li.clone(),
+            signers[3].sign(&li).expect("Unable to sign ledger info"),
+        );
+        partial_signatures.add_signature(signers[3].author(), vote_3.signature().clone());
+
+        let vote_4 = OrderVote::new_with_signature(
+            signers[4].author(),
+            li.clone(),
+            signers[4].sign(&li).expect("Unable to sign ledger info"),
+        );
+
+        assert_eq!(
+            pending_order_votes.insert_order_vote(&vote_0, &verifier, Some(qc.clone())),
+            OrderVoteReceptionResult::VoteAdded(1)
+        );
+
+        vote_0.set_verified();
```

Comment:
what are we trying to do by setting it to verified?

Reply:
Just wanted to make sure that the data structure works when it receives a mix of verified and unverified signatures.